profiles/default/linux/arm/17.0/musl/parent

```diff
@@ -0,0 +1,3 @@
+..
+../../../../../arch/amd64/no-multilib
```

Comment:
Do you really want arm to inherit from amd64?

Reply:
nope, that's a typo